server/src/main/java/keywhiz/KeywhizService.java

```diff
@@ -76,6 +76,18 @@
    * @throws Exception for any uncaught application exception
    */
   public static void main(String[] args) throws Exception {
+    logger.info("Starting Keywhiz...");
+
+    Thread.setDefaultUncaughtExceptionHandler((thread, e) -> {
+      logger.error(
+          String.format(
+              "Uncaught exception in thread %s",
+              thread.getName()),
+          e);
+    });
+
+    System.setSecurityManager(new DisableSystemExitSecurityManager());
```

Comment:
Still experimenting with getting our Keywhiz-killing stack trace into the logs. This time around, we're going to disable System.exit() and also install a global uncaught exception handler.